src/shared-components/theme-engine/theme.json

```diff
@@ -29,6 +31,9 @@
     "fontColorTertiary": "#FFFFFF",
     "fontColorTertiaryHover": "#F5F5F5",
 
+    "fontColorAppMenu": "background: linear-gradient(0deg, #404040, #404040),linear-gradient(0deg, #808080, #808080)",
```

Comment:
i find that it's easier with a theme to restrict it to css *values* rather than full statements. it becomes particularly important when you start allowing users/entities to customize the theme, but it also just makes things more flexible. if we can, let's actually try to restrict the theme to colors for now, and if we need gradients, etc., keep that in the css. (i know the existing theme might have some legacy stuff that breaks this, since we just started with a theme from the old app to get up and running, but since we're starting pretty fresh, let's try to avoid it if possible.)